lib/rows.js

```diff
@@ -160,9 +160,9 @@ function queryRowsOperationImpl(self, builtPlan, streamType, options, bindingArg
     }
   }
 
-  const contentTypeHeader = queryContentType(builtPlan, options.queryType);
+  const contentTypeHeader = graphqlQuery?'application/graphql':queryContentType(builtPlan, options.queryType);
```

Comment:
I think both this and the line below it would benefit from spaces for readability - e.g. 

    const contentTypeHeader = graphqlQuery ? '...' : queryContentType(...);

I also often like doing this for ternary operations:

    const contentTypeHeader = graphqlQuery ? 
        "application/graphql" : 
        queryContentType(builtPlan, options.queryType);

I usually put it on one line if the line is short enough, and then 3 lines if one of the two results is a little lengthy.